core/control-plane/control-plane-aggregate-services/src/test/java/org/eclipse/edc/connector/service/policydefinition/PolicyDefinitionServiceImplTest.java

```diff
@@ -241,6 +253,24 @@ public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
         }
     }
 
+    private static class ValidFilters implements ArgumentsProvider {
+        private static final String PRIVATE_PROPERTIES = "privateProperties";
+        private static final String EDC_NAMESPACE = "https://w3id.org/edc/v0.0.1/ns/";
+        private static final String KEY = "key";
+
+        private static final String VALUE = "123455";
+
+        @Override
+        public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
+            return Stream.of(
+                    arguments(criterion(PRIVATE_PROPERTIES, "=", VALUE)), // skips path element
+                    arguments(criterion(PRIVATE_PROPERTIES + "." + KEY, "=", VALUE)), // skips path element
+                    arguments(criterion(EDC_NAMESPACE + PRIVATE_PROPERTIES + "." + KEY, "=", VALUE)), // skips path element
+                    arguments(criterion(EDC_NAMESPACE + PRIVATE_PROPERTIES + "." + EDC_NAMESPACE + KEY, "=", VALUE)) // skips path element
```

Comment:
those comments are not really explicit... what they mean with path element?

Reply:
done